rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -934,6 +935,16 @@ def test_find_vthor_price(inquirer_defi: 'Inquirer', database: 'DBHandler'):
     assert price.is_close(FVal(0.97656), max_diff=1e-5)
 
 
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+@pytest.mark.parametrize('should_mock_current_price_queries', [False])
+def test_find_morpho_vault_price(database: 'DBHandler', inquirer_defi: 'Inquirer') -> None:
+    """Test that we get the correct price for Morpho vault tokens."""
+    usual_boosted_usdc_vault = create_ethereum_morpho_vault_token(database=database)
+    price = inquirer_defi.find_usd_price(asset=usual_boosted_usdc_vault)
```

Comment:
So as mentioned in discord here if you do a EUR price query it won't work. But this is a deeper error which we have to handle.